features/documentselection/internal/src/main/kotlin/uk/gov/onelogin/criorchestrator/features/documentselection/internal/passport/PassportSelectionScreen.kt

```diff
@@ -0,0 +1,61 @@
+package uk.gov.onelogin.criorchestrator.features.documentselection.internal.passport
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.PreviewLightDark
+import uk.gov.android.ui.theme.m3.GdsTheme
+
+@Composable
+fun PassportSelectionScreen(
```

Comment:
> Similarly, could we use the active voice for naming here?

https://github.com/govuk-one-login/mobile-android-cri-orchestrator/pull/182#discussion_r1995122097

```suggestion
fun SelectPassportScreen(
```

Reply:
Renamed this composable.